packages/flutter/test/material/ink_well_test.dart

```diff
@@ -350,31 +350,29 @@ void main() {
       child: Directionality(
         textDirection: TextDirection.ltr,
         child: Center(
-          child: Focus(
-            focusNode: focusNode,
```

Comment:
Request this node focus can not let the `InkWell` focused, the `InkWell` has its own `Focus` and node.